cloudbuild/presubmit.yaml

```diff
@@ -19,7 +19,7 @@ steps:
       - USERNAME=user
       - SSH_ARGS=--internal-ip --ssh-key-expire-after=1d
       - INSTANCE_NAME=kne-presubmit-$BUILD_ID
-      - INSTANCE_ARGS=--network cloudbuild-workers --image-project gep-kne --image-family kne --machine-type e2-standard-16 --preemptible
+      - INSTANCE_ARGS=--network cloudbuild-workers --image-project gep-kne --image-family kne --machine-type e2-standard-16 --preemptible --scopes=cloud-platform
```

Comment:
What does this flag do?

Reply:
it grants the VM access to the various gcloud APIs. for some reason VMs need a scope and the service account they use needs access to resource